### DISCUSSION TITLE
Simple and tricky case

### DISCUSSION THREAD

**Discussion by aspanin** [vladimir-bukhtoyarov/bucket4j#170] [Q&A]
Hello,

Just simple case: no more than 100 RPS, otherwise the controller is blocked for everyone for 1 minute.
I dug the whole documentation and haven't understood, how to:
- freeze refilling. If a client produces 50 PRS, then after first second we'll get 50 unutilized tokens in bucket, after second one 100 tokens, after third 150 etc. How can I get only 100 tokens per second no matter utilization?
- parking by time. If service got 101 RPS, then client should get HTTP_429 for 1 minute. After it, there're 100 tokens per second again. 

Please, help.

**Comment by vladimir-bukhtoyarov** [comment]
Hello @aspanin 

You are asking about #78 
This feature has not been implemented yet.
